pyproject.toml

```diff
@@ -0,0 +1,74 @@
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+# Check https://flit.readthedocs.io/en/latest/pyproject_toml.html for all available sections
+name = "pyansys"
+version = "2023.1.dev0"
+description = "Pythonic interfaces to Ansys products"
+readme = "README.rst"
+license = "MIT"
+homepage = "https://github.com/pyansys"
+repository = "https://github.com/pyansys/pyansys_landing"
+documentation = "https://docs.pyansys.com/"
+authors = [
+    "ANSYS, Inc. <pyansys.support@ansys.com>",
+]
+maintainers = [
+    "PyAnsys developers <pyansys.maintainers@ansys.com>",
+]
+classifiers = [
+    'Development Status :: 4 - Beta',
+    'Intended Audience :: Science/Research',
+    'Topic :: Scientific/Engineering :: Information Analysis',
+    'License :: OSI Approved :: MIT License',
+    'Operating System :: Microsoft :: Windows',
+    'Operating System :: POSIX',
+    'Operating System :: MacOS',
+    'Programming Language :: Python :: 3.7',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
+]
+
+packages = [
+    { include = "pyansys", from = "src" },
+]
+
+[tool.poetry.dependencies]
+python = ">=3.7,<4.0"
+importlib-metadata = {version = "^4.0", python = "<3.8"}
+ansys-mapdl-core = "==0.63.0"
+ansys-dpf-core = "==0.4.2"
+ansys-dpf-post = "==0.2.2"

```

Comment:
These will have to be updated. As it stands, `pydpf-core==0.5.0` is for 2022R2, which we're still waiting on.

Reply:
At the moment, the latest available one is 0.4.2 (I refer to the released versions). As soon as they publish 0.5.0, dependabot will let us know